ansible/roles/ocp-workload-starter-workshop/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Running Pre Workload Tasks
+  include: ./pre_workload.yml
```

Comment:
`include` is deprecated since ansible 2.4 IIRC.

can you use one of those instead of `include`:
- import_tasks  (static)
- include_tasks (runtime)

prefer static if possible